commitlint.config.js

```diff
@@ -0,0 +1,9 @@
+const { rules } = require('@commitlint/config-conventional');
+
+/** @type {import('@commitlint/types').UserConfig} **/
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'type-enum': [2, 'always', [...rules['type-enum'][2], 'api']],
+  },
+};
```

Comment:
nit: no need for this file since we are not going have `api` stuff in sdk for a reasonable time (IMHO)